ci/launch_cluster.sh

```diff
@@ -13,13 +17,24 @@ CHANNEL="$1"
 VARIANT="$2"
 
 if [ "$VARIANT" == "open" ]; then
-  TEMPLATE="https://s3.amazonaws.com/downloads.dcos.io/dcos/${CHANNEL}/cloudformation/multi-master.cloudformation.json"
+  INSTALLER_URL="https://downloads.dcos.io/dcos/${CHANNEL}/dcos_generate_config.sh"
+  EXTRA_DCOS_CONFIG=""
 else
-  TEMPLATE="https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise-aws-advanced/${CHANNEL}/${VARIANT}/cloudformation/ee.multi-master.cloudformation.json"
+  if [ -z "$DCOS_INSTALLER_TEMPLATE" ]; then
+      echo "Expecting DCOS_INSTALLER_TEMPLATE environment variable"
+      exit 1
+  fi
+  if [ -z "$DCOS_LICENSE" ]; then
+      echo "Expecting $DCOS_LICENSE environment variable"
+      exit 1
+  fi
+  # Replace {} in the template with the channel
+  INSTALLER_URL=${$DCOS_INSTALLER_TEMPLATE/{\}/$CHANNEL}
+  EXTRA_DCOS_CONFIG="  security: ${VARIANT}"$'\n'"  license_key_contents: ${DCOS_LICENSE}"
 fi
 
 echo "Workspace: ${WORKSPACE}"
-echo "Using: ${TEMPLATE}"
+echo "Using: ${INSTALLER_URL}"
 
 PLATFORM=`uname`
```

Comment:
I know it was not your change but shellcheck is saying

```
PLATFORM=`uname`
         ^-- SC2006: Use $(..) instead of legacy `..`.
```

Reply:
Fixed